documentation/commands/deploy.md

```diff
@@ -23,26 +24,24 @@ A few steps are required of the user for a fully functional deployment:
 
   Using Homebrew:
   - `brew install awscli`
```

Comment:
Unrelated to this but I think we can remove this. We currently only need the user to put there credentials file and a link to here: http://docs.aws.amazon.com/cli/latest/userguide/cli-config-files.html should be sufficient.